systemtests/scripts/run_test_kubernetes.sh

```diff
@@ -106,7 +106,6 @@ if [[ -z "$DISABLE_LOG_SYNC" ]]; then
     #stop docker logging
     echo "process for syncing docker logs with PID: ${LOGS_PID} will be killed"
     kill ${LOGS_PID} || true
-    categorize_docker_logs "${DOCKER_LOG_DIR}" || true
```

Comment:
Is this not working anymore?

Reply:
@lulf ut was set implemented in past for categorize logs based on address space (namespace), but we changed deployment to one infra project so address space name is not in pod name so it does not work